lib/developer_portal/lib/liquid/tags/cdn_asset.rb

```diff
@@ -13,45 +13,52 @@ class CdnAsset < Liquid::Tags::Base
 
       def initialize(tag_name, file, tokens)
         super
-        @file = File.join(cdn_host, file).strip # strip because we are allowing the path not enclosed between quotes
+        @file = File.join('/dev-portal-assets', file).strip # strip because we are allowing the path not enclosed between quotes
```

Comment:
Can we set host here as well? 

```suggestion
        @file = File.join(controller.helpers.rails_asset_host_url, '/dev-portal-assets', file).strip # strip because we are allowing the path not enclosed between quotes
```

Reply:
No, we can't, because at this point we don't have access to the `request`, and the request it's required to automatically append the protocol (`https?:`) when it's not provided.

Reply:
This is the point - to hardcode `https`. If somebody so much needs `http`, they can setup the value to include the desired schema.